src/sail_json_backend/json.ml

```diff
@@ -71,6 +71,11 @@ open Ast
 open Ast_defs
 open Ast_util
 
+let debug_enabled =
+  match Sys.getenv_opt "DEBUG_OUT" with
```

Comment:
"DEBUG_OUT" is fairly generic, and I worry about colliding with other uses of a variable with the same name.

Let's add some namespace to the environment variable, calling it "SAIL_DEBUG", perhaps?